config/application.rb

```diff
@@ -2,17 +2,20 @@
 
 require_relative 'boot'
 
+require 'rails'
```

Comment:
If we require rails, do we still need the AR and AC railties required below? I would think not but I've been surprised before.

Reply:
good eye - don't think so.  It came over as a new line from the upgrade;  I have now commented it out.